ext/standard/uniqid.c

```diff
@@ -32,6 +32,7 @@
 #endif
 
 #include "ext/random/php_random.h"
+#include "ext/random/php_random_csprng.h"
```

Comment:
Ditto, is the old header still needed?

Reply:
Yes, also for `php_combined_lcg()` in case the CSPRNG fails [1]. `uniqid()` is a terrible, terrible function that is commonly misunderstood [2]. I hope to deprecate it entirely, because footgun. Probably together with `mt_srand()` et al in a re-spin of https://wiki.php.net/rfc/deprecations_php_8_3#global_mersenne_twister.

[1] It won't on modern OS's, but I did not yet bother to discuss whether the possibility of the CSPRNG failing is something we still want to support or not.
[2] Its initial use-case, based on the output format, is generating the string left of the `@` within a `message-id` email header. It's commonly used for security-sensitive purposes.

Reply:
ACK